DependencyInjection/HttplugExtension.php

```diff
@@ -207,4 +217,20 @@ private function configureAuthentication(ContainerBuilder $container, array $con
                 ->addArgument(new Reference($authServiceKey));
         }
     }
+
+    /**
+     * Verifies that Puli bundle is installed.
+     *
+     * @param ContainerBuilder $container
+     *
+     * @throws \Exception
+     */
+    private function verifyPuliInstalled(ContainerBuilder $container)
+    {
+        if (false === $container->has('puli.discovery')) {
```

Comment:
The order is important because of checking what services has been registered, which needs fallback, etc. But I think it is not a problem then.